Author: ljlane
Descripton: Add test and check targets to make /usr/bin/dh happy.

Index: b/Makefile.am
===================================================================
--- a/Makefile.am	2009-04-06 14:36:50.302284885 -0400
+++ b/Makefile.am	2009-04-06 14:36:55.451284922 -0400
@@ -123,3 +123,6 @@
 # Using if..fi avoids an ugly "error (ignored)" message :)
 install-exec-hook:
 	-if test -z "${DESTDIR}"; then /sbin/ldconfig; fi;
+
+test check:
+	echo "Are we on the air? Hello, Mom!"
